examples/server/server.c

```diff
@@ -2473,7 +2473,7 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
 #ifndef NO_PSK
         const char *defaultCipherList = cipherList;
 
-        SSL_CTX_set_psk_server_callback(ctx, my_psk_server_cb);
+        wolfSSL_CTX_set_psk_server_callback(ctx, my_psk_server_cb);
```

Comment:
Please revert this line. The example server is supposed to use the openssl compatibility API's (when one exists). Historically the server example is supposed to be based on openssl compatibility API's. It has moved a bit away from this now.

Reply:
Fixed it.  But I have to say its somewhat jarring to read because `wolfSSL_`  is used just below it.